src/frontend/src/eth/components/loaders/LoaderMultipleEthTransactions.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { debounce } from '@dfinity/utils';
+	import { enabledEthereumTokens } from '$eth/derived/tokens.derived';
+	import { loadTransactions } from '$eth/services/transactions.services';
+	import { enabledErc20Tokens } from '$lib/derived/tokens.derived';
+
+	const load = async () => {
+		await Promise.allSettled(
```

Comment:
Just a debounce to prevent multiple load triggers feels a bit light. Did you test to ensure that it effectively prevents unnecessary loads?

Reply:
you are right! it was a very lax measure: so, I make it run only when the ERC20 tokens store is initialized. I included more tests to verify that it is not called more than once if the stores are not loaded.